### PR TITLE
Feature/repo 5118 node download event

### PR DESCRIPTION
**Pull request by SaraAspery**

### CODE REVIEW

File: src/main/java/org/alfresco/repo/node/AbstractNodeServiceImpl.java [hunk 1]
```diff
@@ -1,28 +1,28 @@
-/*
```
Comment (jamalkm) [inline src/main/java/org/alfresco/repo/node/AbstractNodeServiceImpl.java:1]: There are a lot of whitespace changes in this file, can't we just revert it back to its original format?

File: .travis.yml [hunk 2]
```diff
@@ -18,6 +18,7 @@ branches:
   only:
     - master
     - /support\/.*/
+    - feature/REPO-5118_Node_Download_Event
```
Comment (jamalkm) [inline .travis.yml:21]: Please remove all the temp Dev release stuff.

File: src/main/java/org/alfresco/repo/node/AbstractNodeServiceImpl.java (hunk 1 above)
Comment (cmshields) [reply src/main/java/org/alfresco/repo/node/AbstractNodeServiceImpl.java]: I have reverted the whitespace changes in this file.

File: .travis.yml (hunk 2 above)
Comment (cmshields) [reply .travis.yml]: Dev release stuff has been removed.

**Pull request merged**